em_plugins/webview/.eslintignore

```diff
@@ -0,0 +1,3 @@
+build
+dist
+example-app
```

Comment:
`example-app` appears to be leftover from the starter code.

Reply:
Yes, I'll remove it. It auto generates when using a cli to create it from scratch.